scss/main.scss

```diff
@@ -0,0 +1,5 @@
+@import url('https://fonts.googleapis.com/css2?family=PT+Sans+Narrow&display=swap');
```

Comment:
Personally, I'd move this font to the mail file, or to a file specifically for fonts. But I'll leave that at your discretion. Great work!